README.rst

```diff
@@ -387,6 +386,19 @@ The following object storage types are supported:
  * ``container_name`` for the name of Azure Storage container used to store
    objects
 
+* ``swift`` for OpenStack Swift, required configuration keys:
+
+ * ``user`` for the Swift user ('subuser' in Ceph RadosGW)
+ * ``key"`` for the Swift secret_key
```

Comment:
What is this secret key?  I can't find this concept in the OpenStack documentation.

I do find something in the Swift Ceph emulation, but is this the user's password?